internal/deploy/parameters.go

```diff
@@ -312,6 +331,43 @@ func (c *ChangeSet) getBuilder(spec ketchv1.AppSpec) string {
 	return *c.builder
 }
 
+func (c *ChangeSet) getUnits() (int, error) {
+	if c.units == nil {
+		return 0, nil
+	}
+	if *c.units < 1 {
```

Comment:
What do we think about permitting setting units to 0, essentially stopping the App? Crazy?

Reply:
Kubernetes doesn't support stopping/pausing as far as I know, but I think setting replicas (pods) to 0 is a way to kind of get the same effect. As far as this pr is concerned, I think that it is a bit out of scope since it will require some more discussion.